dydx/dydxPresenters/dydxPresenters/_v4/TakeProfitStopLoss/dydxTakeProfitStopLossViewPresenter.swift

```diff
@@ -53,13 +53,24 @@ private class dydxTakeProfitStopLossViewPresenter: HostedViewPresenter<dydxTakeP
         super.start()
 
         guard let marketId = marketId else { return }
+        AbacusStateManager.shared.triggerOrders(input: marketId, type: .marketid)
+
+        Publishers
+            // note we use a zip here intentionally so that the user input is not overwritten unless triggerOrders updates
+            // which should not really happen unless the active trigger order(s) get triggered or they are placed simultaneously
+            // on another platform
+            .Zip(AbacusStateManager.shared.state.selectedSubaccountPositions,
+                 AbacusStateManager.shared.state.selectedSubaccountTriggerOrders)
+            .sink { [weak self] subaccountPositions, triggerOrders in
+                self?.update(subaccountPositions: subaccountPositions, triggerOrders: triggerOrders)
+            }
+            .store(in: &subscriptions)
```

Comment:
splitting out this publisher fixes the overwriting of fresh user input in TP/SL fields